diff --git a/bhordesgame.go b/bhordesgame.go
index e8f69ca..dbeb23b 100644
--- a/bhordesgame.go
+++ b/bhordesgame.go
@@ -28,7 +28,7 @@ func Ignore[T any](t T, e error) T {
 
 var availableLangs = []language.Tag{language.English, language.French}
 
-//go:embed lang templates/*
+//go:embed lang gen
 var f embed.FS
 
 func main() {
@@ -43,15 +43,14 @@ func main() {
 		"incr":       func(i int) int { return i + 1 },
 		"decodeGoal": decodeGoal,
 		"mkmap":      mkmap,
-	}).ParseFS(f, "templates/*.html")))
-	r.Static("/style", "style")
-	r.Static("/script", "script")
+	}).ParseFS(f, "gen/templates/*.html")))
+	r.Static("/style", "gen/style")
+	r.Static("/script", "gen/script")
 	r.StaticFile("/favicon.ico", "asset/favicon.ico")
 	r.StaticFile("/question.svg", "asset/question.svg")
 
 	loadTranslations(f, availableLangs)
 	lngHandler := languageSelector(availableLangs)
-	lngHandler = func(ctx *gin.Context) { ctx.Set(LNG_KEY, "@@{ISO639-1}") }
 
 	r.POST("/", connectionHandle)
 	r.GET("/", lngHandler, indexHandle)
